### PR TITLE
RULEAPI-681 Fix `sqKey` (stop using legacy key where it's relevant)

### PR DESCRIPTION
**Pull request by vilchik-elena**

### CODE REVIEW

**Review by arseniy-sonar — APPROVED** [review]
I trust that this change will not break our analyzers. I validated, that the top-level metadata.json have no more legacy keys for `skKey` field.

File: rules/S1514/javascript/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "title": "\"eval\" and \"arguments\" should not be bound or assigned",
   "type": "BUG",
-  "status": "deprecated",
```
[the remark this answers is not in the document]
Comment (vilchik-elena) [reply rules/S1514/javascript/metadata.json:4]: indeed...  I will do it in another PR

**Comment by arseniy-sonar** [comment]
BTW, does it fix RULEAPI-681?

**Comment by arseniy-sonar** [comment]
Just checked, it does fix :tada:

**Review by arseniy-sonar — APPROVED** [review]

**Pull request merged**